src/v2/view-builder/views/captcha/CaptchaView.js

```diff
@@ -99,15 +94,13 @@ export default View.extend({
     document.getElementById(Enums.WIDGET_CONTAINER_ID).appendChild(scriptTag);
   },
 
-  /**
-   *  Parse through the uiSchema to extract the field that has the CAPTCHA hint associated with it
-  * */ 
-  _getFieldWithCaptchaHint() {
-    const uiSchema = this.options.currentViewState.uiSchema || [];
-    for (const schema of uiSchema) {
-      if (schema.hint === Enums.HINTS.CAPTCHA) {
-        return schema.name;
-      }
-    }
-  },
+  _addHCaptchaFooter() {
+    // NOTE: insetAdjacentHTML() is supported in all major browsers: 
+    // https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML#browser_compatibility
+    document.getElementsByClassName(WIDGET_FOOTER_CLASS)[0].insertAdjacentHTML('beforeend',
```

Comment:
In theory, `(WIDGET_FOOTER_CLASS)[0]` could be null hence cause NPE.
Let's add safe check.

Reply:
Yep that's true, I'll add a safeguard.